.changeset/fifty-laws-hammer.md

```diff
@@ -0,0 +1,5 @@
+---
+'sku': patch
```

Comment:
I would at least make it a `minor`.

My rule of thumb is:
- if a dep upgrade is a major, my bump is a major or minor (it depends™️)
- if a dep upgrade is a minor, my bump is a patch
- if a dep upgrade is a patch, my bump is a patch

Reply:
But that's so arbitrary. A downstream dep could have 0 effect on the API and it'd still be a minor (basically this PR).

Reply:
There are projects with `dangerouslySetEslintConfig` and custom ESLint plugins.

Reply:
Sure, but that's a liability they've taken on themselves. This doesn't really add any features to sku, but it provides access to eslint 8's new features I guess. I guess we're indirectly supporting new things, so maybe it does make sense to be a `minor`. Though I don't agree with picking the upgrade based on the dependency upgrade, that just feels too arbitrary if it has no effect on consumers.